New settings page for the daily spending wallet (#898)
BitcoinDesign/Guide: guide/daily-spending-wallet/settings.md

      modalImage: bitcoin-unit-big
      alt: Mobile screen with automatic, bitcoin, and satoshi options
      caption: The bitcoin unit can also be switched contextually in the send screen.
images_security:
    - file: security
      modalImage: security-big
      alt: Mobile screen with PIN, FaceID, wallet deletion and wallet limit options
      caption: Options to limit unwanted access and payments.
    - file: delete-wallet
      modalImage: delete-wallet-big
      alt: Confirmation modal for deleting the wallet from the device
      caption: This screen requires very clear language to ensure users understand the implications.
images_network:
    - file: network
      modalImage: network-big
      alt: Mobile screen with bitcoin and lightning network, as well as Tor options
      caption: Overview screen for network options.
    - file: bitcoin-network
      modalImage: bitcoin-network-big
      alt: Mobile screen with bitcoin network customization options
      caption: Users can choose their own node to connect to.
    - file: lightning-network
      modalImage: lightning-network-big
      alt: Mobile screen with lightning network customization options

>       caption: Information about the users node, and additional security options.

Reviewer: ```suggestion
      caption: Information about the user’s node, and additional security options.
```

    - file: watchtower
      modalImage: watchtower-big
      alt: Mobile screen with lightning network watchtower options
      caption: Users can enable watchtower functionality to protect their funds.
---

{% include picture.html
    image = "/assets/images/guide/daily-spending-wallet/settings/settings.jpg"
    retina = "/assets/images/guide/daily-spending-wallet/settings/settings@2x.jpg"
    mobile = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile.jpg"
    mobileRetina = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile@2x.jpg"
    alt-text = "An oil painting of a big, clunky machine with a big bitcoin gear"

>     caption = "How AI thinks Matisse would have visualized bitcoin settings."

Reviewer: Is this a DALLE creation?

Reviewer: Not a change request, just curious.

    width = 1600
    height = 600
    layout = "full-width"
%}

<!--

This is more of a summary page that bring together all the application settings that
the user encounters throughout the app experience into a single, concise form.

-->

# Settings
{: .no_toc }

---

<div class="glossary-toc" markdown="1">
 * Table of contents
{:toc}
</div>

---

<div class="center" markdown="1">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/settings/overview.png"
   retina = "/assets/images/guide/daily-spending-wallet/settings/overview@2x.png"
   modalImage = "/assets/images/guide/daily-spending-wallet/settings/overview-big.png"
   alt-text = ""
   width = 250
   height = 541
   layout = "float-right-desktop -background -shadow"

>    caption = "The overview screen provides an easy-to-understand grouping of settings."

Reviewer 2: Minor nit but having a break line at top and bottom of the list looks less clean then omitting them

%}

As you probably noticed in previous pages, various settings can be configured contextually throughout the app experience (like [hiding and unhiding the balance]({{ '/guide/daily-spending-wallet/privacy/#hiding-sensitive-information' | relative_url }})). All these options, and a few extra ones, are bundled in the settings area.

While not the most frequently visited part of an application, it fills an important functional role by giving users precise control over their experience.

</div>

## General

> This sub-section includes various non-essential settings that are not covered in the other categories and let users adjust the application to their context and personal preferences.

Author: It's referring to `settings`, so isn't the plural `let` correct?

> {% include image-gallery.html pages = page.images_general %}

Reviewer 2: Notifications option isn't cantered properly in first screen

What does incoming transactions toggle mean? No context to it.

The "All options" looks like it could be selected in the second image when its a title.

Author: `Notifications` is a title, there's intentional spacing above. Made it semi-bold instead of medium to make it more clear. Also updated the other titles, including `All options`.

Reviewer 2: Removing the break line under the title would make it a lot clearer

Author: Lines removed.

## Fees

<div class="center" markdown="1">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/settings/fees.png"
   retina = "/assets/images/guide/daily-spending-wallet/settings/fees@2x.png"

>    modalImage = "/assets/images/guide/daily-spending-wallet/settings/fees-big.png"

Reviewer 2: Notifications option isn't cantered properly in first screen

What does incoming transactions toggle mean? No context to it.

Reviewer 2: The "All options" looks like it could be selected in the second image when its a title.

   alt-text = ""
   width = 250
   height = 541
   layout = "float-right-desktop -background -shadow"
   caption = "The fees screen provides a clear overview of all fees that may be charged."
%}

> Paying fees for transactions is unavoidable and typically not something users appreciate. Being transparent and communicating them clearly in context (e.g. when [reviewing a payment]({{ '/guide/daily-spending-wallet/sending/#review--approval' | relative_url }}) before sending it) is crucial. Since contextual information is selective, this fee screen in settings is needed to provide a concise summary of all the types of fees that may be charged.

Reviewer 2: Not appreciate sounds off, users expect fees in many situations

Author: Do you like paying fees? Expecting and appreciating are different.

Reviewer: You know, it didn't bother me when I read it, but I can see why it's a confusing sounding word for Bosch.

The problem is that the word "appreciate" in English can have 2 meanings -- essentially, it can mean to _enjoy_ or it can mean to _recognize_ or _understand_ something. I do not appreciate paying fees, but I do appreciate the need for fees and the role they play in a transaction. Same word, different meaning in a different context.

@GBKS It's not a blocker for me, but I think if you swapped it out with the word "enjoy" that would be unambiguous and undebatable.

Author: Good point. Changed to `enjoy`.

Reviewer 2: Interesting point, didn't know that. Is that a UK / US English thing? (us auzzies do UK english fyi)

During regular use, only lightning network fees will apply to transactions in this wallet. An additional fee may be charged for opening new channels to increase liquidity.

> When users send to an on-chain address, a swap fee applies. This fee also applies if the user receives to an on-chain (swap) address. This may cause confusion, as typically the sender is reponsible for paying fees.

Reviewer: ```suggestion
When users send to an on-chain address, a swap fee applies. This fee also applies if the user receives to an on-chain (swap) address. This may cause confusion, as typically, the sender is responsible for paying fees.
```

Reviewer 2: This also only occurs if the business charges for that fee otherwise it's just a normal on-chain transascton fee.

May want to add a sentence or two about fees that wallet providers charge (non-network ones) and how these should also be clearly communicated.

Author: How would you word this?

</div>

## Privacy

<div class="center" markdown="1">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/settings/privacy.png"
   retina = "/assets/images/guide/daily-spending-wallet/settings/privacy@2x.png"
   modalImage = "/assets/images/guide/daily-spending-wallet/settings/privacy-big.png"
   alt-text = ""
   width = 250
   height = 541
   layout = "float-right-desktop -background -shadow"

>    caption = "Privay options focus on the display of sensitive data."

Reviewer: ```suggestion
   caption = "Privacy options focus on the display of sensitive data."
```

%}

[Privacy]({{ '/guide/daily-spending-wallet/privacy/' | relative_url }}) settings are straightforward and focused on whether balances are visible or hidden. While there are some best practices for users, much of the responsibility to ensure good privacy rests in the hands of the developers.

</div>

## Security

> [Security]({{ '/guide/daily-spending-wallet/security/' | relative_url }}) is all about preventing unwanted access to the application. Basic access options (PIN, FaceID) are configured during [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) and users receive regular reminders to review them. Wallet limits are an additional option, only available in settings, to block potentially unwanted high-value payments.

Reviewer: ```suggestion
[Security]({{ '/guide/daily-spending-wallet/security/' | relative_url }}) is all about preventing unwanted access to the application. Basic access options (PIN, FaceID) are configured during [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}), and users receive regular reminders to review them. Wallet limits are an additional option, only available in settings, to block potentially unwanted high-value payments.
```

{% include image-gallery.html pages = page.images_security %}

## Wallet backup

<div class="center" markdown="1">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup.png"
   retina = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup@2x.png"
   modalImage = "/assets/images/guide/daily-spending-wallet/settings/wallet-backup-big.png"
   alt-text = ""
   width = 250
   height = 541
   layout = "float-right-desktop -background -shadow"
   caption = "Backup options are typically configured in first use and rarely need to be accessed."
%}

> [First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) also guides users towards appropriate [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) options, and users may never need to access this screen. A unique piece of information here is when the last cloud backup was performed. This can give users peace of mind that their information is securely backed up.

Reviewer 2: ```suggestion
[First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) also guides users towards appropriate [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) options, and users may never need to access this screen. A unique piece of information here is when the last cloud backup was performed. This can give users peace of mind that their information is securely being backed up.
```

As it occurs frequently

</div>

## Network

> Most users will not need or want to change network settings, but it is important to offer these features for those who do. While it may be personal preference for some, it may be a privacy or security matter for others to connect through services they personally maintain or trust. Interoperability is a core [design principle]({{ '/guide/getting-started/principles/#interoperability' | relative_url }}), and also makes the application more versatile.

Reviewer: ```suggestion
Most users will not need or want to change network settings, but it is important to offer these features for those who do. While it may be personal preference for some, it may be a privacy or security matter for others to connect through services they personally maintain or trust. Interoperability is a core [design principle]({{ '/guide/getting-started/principles/#interoperability' | relative_url }}) and also makes the application more versatile.
```

> {% include image-gallery.html pages = page.images_network %}

Reviewer 2: Image 2 - what is the Bitcoin node (BIP 157) option representing? Is that the node that is connected to when default is selected? Shouldn't default also be on if its the default.

Author: Good point. Updated to be similar to the watchtower copy. When you disable the default setting, then the input should become editable.

## Help & support

<div class="center" markdown="1">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/settings/help.png"
   retina = "/assets/images/guide/daily-spending-wallet/settings/help@2x.png"
   modalImage = "/assets/images/guide/daily-spending-wallet/settings/help-big.png"
   alt-text = ""
   width = 250
   height = 541
   layout = "float-right-desktop -background -shadow"
   caption = "Help & support options ensure users can get the right information when they need it."
%}

> Open-source projects do not have large support teams, but instead often rely on their communities and forum discussion to allow users to help each other. Contributors are also part of this activity, but usually cannot dedicate time to address all support needs. Therefore it is important to provide several different methods to get help, it distributes the burden of support across documentation, community, and contributors.

Reviewer: ```suggestion
Open-source projects do not have large support teams but instead often rely on their communities and forum discussion to allow users to help each other. Contributors are also part of this activity but usually cannot dedicate time to address all support needs. Therefore it is important to provide several different methods to get help. It distributes the burden of support across documentation, community, and contributors.
```
